providers/os/resources/vulnmgmt.go

```diff
@@ -133,13 +138,18 @@ func (v *mqlVulnmgmt) populateData() error {
 
 	mqlVulnCves := make([]interface{}, len(vulnReport.Cves))
 	for i, c := range vulnReport.Cves {
-		parsedPublished, err := time.Parse(time.RFC3339, c.PublishedAt)
+		var parsedPublished time.Time
```

Comment:
I think this should be a time pointer and we pass in MQL the pointer, so that the value is properly null.

Reply:
Good point. Changed it and now the dates show up as null, when empty.